pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -397,3 +484,33 @@ func (p *staticPolicy) generateCPUTopologyHints(availableCPUs cpuset.CPUSet, req
 
 	return hints
 }
+
+func (p *staticPolicy) isContainerMapSynced(s state.State) bool {
```

Comment:
this probably deserves a comment explaining what it does and what determines the return value